templates/unifi-poller.xml

```diff
@@ -31,110 +29,105 @@ https://github.com/davidnewhall/unifi-poller/wiki/Installation</Description>
   <Environment>
+    <!-- poller -->
     <Variable>
-      <Value/>
-      <Name>UP_MODE</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_POLLER_DEBUG</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_INFLUX_USER</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_POLLER_QUIET</Name>
     </Variable>
+    <!-- unifi -->
     <Variable>
-      <Value/>
-      <Name>UP_INFLUX_URL</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_URL</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_INFLUX_DB</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_USER</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_UNIFI_USER</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_PASS</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_UNIFI_PASS</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_SAVE_DPI</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_UNIFI_URL</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_SITES_0</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_DEBUG</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_SAVE_IDS</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_INTERVAL</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_SAVE_SITES</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_INFLUX_PASS</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_UNIFI_CONTROLLER_0_VERIFY_SSL</Name>
     </Variable>
+    <!-- influx -->
     <Variable>
-      <Value/>
-      <Name>UP_HTTP_LISTEN</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_INFLUXDB_DISABLE</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_SITES</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_INFLUXDB_URL</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_SAVE_IDS</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_INFLUXDB_USER</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_SAVE_SITES</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_INFLUXDB_PASS</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_REAUTHENTICATE</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_INFLUXDB_DB</Name>
     </Variable>
     <Variable>
-      <Value/>
-      <Name>UP_VERIFY_SSL</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_INFLUXDB_INTERVAL</Name>
     </Variable>
+    <!-- prometheus -->
     <Variable>
-      <Value/>
-      <Name>UP_QUIET</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_PROMETHEUS_DISABLE</Name>
+    </Variable>
+    <Variable>
+      <Value></Value>
+      <Name>UP_PROMETHEUS_HTTP_LISTEN</Name>
     </Variable>
   </Environment>
```

Comment:
You can [shave off the xml](https://selfhosters.net/docker/templating/templating/#114-shave-off-the-xml)

Reply:
oh shoot, those aren't actually needed? That sure makes it nicer to look at.